tests/test_PolarBasis2D.py

```diff
@@ -213,6 +213,16 @@ def testPolarBasis2DAdjoint(self):
         # Namely, if A = evaluate, B = evaluate_t, and B=A^t, we will have
         # (y, A*x) = (A^t*y, x) = (B*y, x)
         x = np.random.randn(self.basis.count)
+
```

Comment:
I think we do this transformation inside the `evaluate`  function. Should we remove this recalculation based on the conjugate part?

Reply:
I'm not sure which transformation you're referring to. Line 219 below? We need this for the inner product `np.dot(y_t, x)` to be correct. Line 226 calls `evaluate`, which indeed symmetrizes `x` (just like line 219). However, when `x` is used in `np.dot(y_t, x)`, that `x` will not be symmetrized. The underlying issue is that `evaluate` does not operate on the entire space of polar Fourier coefficients, only on the subspace of Hermitian symmetric coefficients (since these correspond to real images). For the FB basis, we don't have this problem, because we explicitly break the symmetry by only storing half the frequencies. It might make sense to do this here as well, but currently the common lines assumes that all the frequencies are calculated, so that would break if we only give it half (although it's possible to rewrite it to only take half).

Reply:
OK. I understand. If we use the output from `evalute_t` as `x,` then we don't need do this.

Reply:
Correct, that would work as well.

Reply:
Please approve the PR if you think this is the last issue. I will submit a new PR for the orientation code which depends on this one.